data/arrow-ipc-stream/integration/README.md

```diff
@@ -0,0 +1,10 @@
+# Arrow IPC Integration "Gold" Files
+
+These files contain data generated by well-known versions and implementations
+of Apache Arrow. Unless otherwise specified, the implementation is Arrow C++
+and the data is little-endian.
```

Comment:
I think that little-endian is reasonable but how about specify implementation explicitly?

We may not use big-endian but we may use other implementation such as Rust and Go.